mots.csl

```diff
@@ -0,0 +1,299 @@
+    <title>Mots (French)</title>
+    <title-short>Mots</title-short>
+    <id>http://www.zotero.org/styles/mots</id>
+    <link href="http://www.zotero.org/styles/mots" rel="self"/>
+    <link href="http://www.zotero.org/styles/vita-latina" rel="template"/>
+    <link href="https://journals.openedition.org/mots/76" rel="documentation"/>
+    <author>
+      <name>Louis Autin (SU)</name>
+      <email>louis.autin@sorbonne-universite.fr</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="humanities"/>
+    <issn>0243-6450</issn>
+    <eissn>1960-6001</eissn>
+    <summary>Zotero stylesheet for Mots (journal)</summary>
+    <published>2022-09-02T00:00:00</published>
+    <updated>2022-09-02T00:00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <!-- INSTRUCTIONS DE FORMATAGE EN FRANCAIS -->
+  <locale>
+    <terms>
+      <term name="editor" form="short">
+        <single>éd.</single>
+        <multiple>éd.</multiple>
+      </term>
+      <term name="translator" form="verb-short">trad.</term>
+      <term name="in">dans</term>
+      <term name="no date">pas de date</term>
+    </terms>
+  </locale>
+  <!-- MACROS DE CITATION -->
+  <macro name="auteurs-court">
+    <names variable="author">
+      <name form="short" initialize-with=". " and="text" delimiter=", "/>
+      <et-al font-style="italic"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text macro="titre"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="indication-localisation">
+    <group delimiter=", ">
+      <choose>
+        <if type="book report thesis" match="any">
+          <group>
+            <text term="volume" form="short" suffix=" "/>
+            <number variable="volume" form="numeric"/>
+          </group>
+        </if>
+      </choose>
+      <group delimiter=" ">
+        <choose>
+          <if locator="page" match="none">
+            <label variable="locator" form="short" suffix=" "/>
+          </if>
+        </choose>
+        <text variable="locator" prefix="p. "/>
+      </group>
+    </group>
+  </macro>
+  <!-- MACROS DE BIBLIOGRAPHIE -->
+  <macro name="contributeurs-tri">
+    <names variable="author">
+      <name name-as-sort-order="all" and="text" sort-separator=" " delimiter=", " delimiter-precedes-last="never"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="auteurs">
+    <names variable="author">
+      <name name-as-sort-order="all" and="text" delimiter=" " delimiter-precedes-last="never" sort-separator=" ">
+        <name-part name="family" font-variant="small-caps"/>
+      </name>
+      <label form="short" prefix=" (" suffix=")"/>
+      <substitute>
+        <names variable="editor">
```

Comment:
why are you using these elaborate substitutes instead of just `<names variable="editor"/>` (the other elements are inherited from author)